docs/source/boards/esp8266.rst

```diff
@@ -0,0 +1,201 @@
+
+	**$ esptool**
+
+Connect the ESP8266 to your computer. Take the USB cable from the kit and Put the USB-C in one of your ports and then connect the micro-USB to the microcontroller.
+
+Succeeding, go to the folder where you installed Micropython firmware. Use **$ls** command to list files and directories and **$ cd** to change the current working directory.
+
+Type **$ dmesg** to see the port, you should have a similar output:
+
+.. image:: images/esp8266/verify_dmesg.png
+	:align: center
+
+In this case the port is *ttyUSB0*.
+
+After, use the command:
+
+	**$ esptool.py --port /dev/ttyUSB0 erase_flash**
+
+for erasing the flash memory on the board. Instead of *ttyUSB0* you might have another port. You have to put the one that you have seen earlier.
+
+Press the reset (RST) button from your ESP8266, then use the command: 
+
+	**$ esptool.py --port /dev/ttyUSB0 --baud 460800 write_flash --flash_size=detect 0 esp8266-20170108-v1.8.7.bin**
+
+Keep in mind to put the port that you used in the previous command and pay attention to the version of Micropython that you have installed. Instead of *“esp8266-20170108-v1.8.7.bin”* you might have another version. You must replace it in the command. If you encounter errors, you need to reduce the baud rate (for example 115200 or up down).
+
+Next, connect to the serial console with command:
+
+	**$ screen /dev/ttyUSB0 115200**
+
+“115200” is the speed read by serial port. To close it type **Ctrl+D** or **Ctrl+a** followed by **Ctrl+\\**.
+
+**Congratulations**, now you have Micropython on your ESP8266!
+
+|
+
+3. Install and configure Micropython on MacOS
+---------------------------------------------
+
+3.1 Download Micropython firmware
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Open a browser and type this link: https://micropython.org/download/esp8266/,
+then go install the latest version (without opening it). 
+
+*Suggestion*: create a folder on your computer named “esp8266” or “micropython” and download it there.
+
+.. image:: images/esp8266/mp_firmware.png
+	:align: center
+
+3.2 Install NodeMCU PyFlasher and flashing Micropython on ESP8266
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+NodeMCU PyFlasher is a new GUI tool to flash NodeMCU based on esptool.py and wxPython.
+
+First, you have *to connect the ESP8266 to your computer*. Take the USB cable from the kit and Put the USB-C in one of your ports and then connect the micro-USB to the microcontroller.
+
+Check this link `install NodeMCU <https://github.com/marcelstoer/nodemcu-pyflasher/releases>`_, if you have High Sierra. Scroll down to the executables and click on the third executable (*NodeMCU-PyFlasher-4.0.dmg*).
+
+After the installation will be completed, click on the NodeMCU PyFlasher .exe file and you should have a similar window: 
+
+.. image:: images/esp8266/nodeMCU_mac.png
+	:align: center
+
+First, you have to choose your serial port, in this case the port is: “/dev/cu.SLAB_USBtoUART”
```

Comment:
Yopu might be missing a subtitle here like "Flashing Micropython"?

Reply:
It's the same subtitle structure as Windows "Install NodeMCU PyFlasher and flashing Micropython on ESP8266". 
Should I divide them like "Install NodeMCU PyFlasher" and " Flashing Micropython on ESP8266" ?

Reply:
Got it